src/docs/product/crons/troubleshooting.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Troubleshooting
+sidebar_order: 3
+---
+
+## Cron monitor isn't notifying me of failures
+
+If your cron monitor isn't sending any notifications, ensure that your alert rule is set up with the appropriate [filters](/product/crons/alerts/#step-1-set-filters) and [alert rule actions](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions)
```

Comment:
```suggestion
If your cron monitor isn't sending any notifications, make sure that your alert rule is set up with the appropriate [filters](/product/crons/alerts/#step-1-set-filters) and [alert rule actions](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions).
```

I would add the feedback section here rather than have it live on a separate page.